makefile

```diff
@@ -36,6 +36,16 @@ SHELL := /bin/bash
 # Launch zipkin.
 # http://localhost:9411/zipkin/
 
+
+# ==============================================================================
+# Install dependencies
+
+dev.setup.mac:
+	brew update
```

Comment:
I realize now why we need to run update, to make sure we get the latest version of kid and kustomize available. We need to add kubectl to this list as well.